pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -120,6 +123,7 @@ def __init__(
         # options for MPS
         self._max_bond_dim = kwargs.get("max_bond_dim", None)
         self._cutoff = kwargs.get("cutoff", np.finfo(self._c_dtype).eps)
+        self._contract = kwargs.get("contract", "auto-mps")
```

Comment:
Should `contract="auto-mps"` be a valid option to pass to the device for execution? I get an error:

```python
from pennylane_lightning.lightning_tensor import LightningTensor

@qml.qnode(LightningTensor(contract="auto-mps"))
def circuit():
    qml.X(0)
    return qml.expval(Z(0))

circuit()
```

```pycon
ValueError: Option `contract` should be one of {False, True, 'swap-split-gate', 'auto-split-gate', 'split-gate', 'reduce-split', 'split'}, but got 'auto-mps'.
```

Same for `"nonlocal"`, works for `True`.

Reply:
We need the functionality introduced in `quimb 1.8.1`, so we need to install it as `pip install -U git+https://github.com/jcmgray/quimb.git` until it is officially released. See [this discussion](https://github.com/jcmgray/quimb/issues/230#issuecomment-2091511588) or the [1.8.1 changelog](https://quimb.readthedocs.io/en/latest/changelog.html#v1-8-1-unreleased) directly. If you run this with `quimb 1.8.1` it works fine